src/Tizen.NUI.Components/Controls/Pagination.cs

```diff
@@ -476,7 +476,9 @@ private void CreateIndicator()
                 MixColor = (indicatorColor == null) ? new Color(1.0f, 1.0f, 1.0f, 0.5f) : indicatorColor,
                 Opacity = (indicatorColor == null) ? 0.5f : indicatorColor.A
             };
-            indicator.Position = new Position2D((int)(paginationStyle.IndicatorSize.Width + paginationStyle.IndicatorSpacing) * indicatorList.Count, 0);
+            Position2D indicationPos = new Position2D((int)(paginationStyle.IndicatorSize.Width + paginationStyle.IndicatorSpacing) * indicatorList.Count, 0);
+            indicator.Position = indicationPos;
+            indicationPos.Dispose();
```

Comment:
I think the warning is caused because there was an implicit type conversion from `Position2D` to `Position`.
So just changing `Position2D` to `Position` in the original code will fix the warning.

Reply:
The indicator.Position is  Vector2, so cannot change Position2D to Position.

Reply:
So then please make it Vector2.
```
indicator.Position = new Vector2((int)(paginationStyle.IndicatorSize.Width + paginationStyle.IndicatorSpacing) * indicatorList.Count, 0);
```

Reply:
Done